### PR TITLE
added collection support for comma separated values

### PR DESCRIPTION
**Pull request by dpetrovaliev**
Fixes issue #69

### CODE REVIEW

File: tests/FeatureModelCastTest.php [hunk 1]
```diff
@@ -96,6 +96,39 @@ public function it_saves_a_null_value_for_nullable_array_of_enums()
...
+            'status' => StatusEnum::draft(),
+            'set_of_enums' => [StatusEnum::draft(), StatusEnum::archived()],
+        ]);
+
+        $model->refresh();
+
+        $this->assertTrue($model->status->equals(StatusEnum::draft()));
+        $this->assertEquals('draft', $model->getOriginal('status'));
+        $this->assertIsArray($model->set_of_enums);
+        $this->assertCount(2, $model->set_of_enums);
```
Comment (Gummibeer) [inline tests/FeatureModelCastTest.php:127]: An assertion to the raw attribute is missing. Right now this test would also pass if the JSON formater is applied. So it should check that the raw attribute is `DRAFT,ARCHIVED` and in the best case add a similar assertion to one of the JSON ones.

**Review by Gummibeer — CHANGES_REQUESTED** [review]
Thanks for your work! 🥳
In case you don't like the `comma` option name I'm open to suggestions `CSV` would be also an idea I had.
But I would like to use it as a format in the same cast and don't introduce a new cast.

File: tests/FeatureModelCastTest.php (hunk 1 above)
Comment (dpetrovaliev) [reply tests/FeatureModelCastTest.php:127]: good point will add the assertion shortly

Comment (dpetrovaliev) [reply tests/FeatureModelCastTest.php:128]: it's done and waiting for your review

Comment (dpetrovaliev) [reply tests/FeatureModelCastTest.php:128]: Hey, @Gummibeer hope you're doing well! Have you been able to check the PR?

**Comment by dpetrovaliev** [comment]
Hi, @Gummibeer
I haven't heard anything from you for a while. Is there a chance for this week to release a new version with this PR?

**Review by Gummibeer — APPROVED** [review]

**Comment by Gummibeer** [comment]
Hey,
sorry for the delay - lot of work to do. Looks good now. Will merge it and release in a few minutes!
Thanks for your kind reminder! Yes I'm fine. 🙂

**Comment by Gummibeer** [comment]
https://github.com/spatie/laravel-enum/releases/tag/2.5.0

**Pull request merged**